### PR TITLE
Several FITS keyword fixes

### PR DESCRIPTION
**Pull request by jluhrs**

### CODE REVIEW

File: modules/server_new/src/main/scala/observe/server/gws/package.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package observe.server
+
+import coulomb.*
+import coulomb.define.*
+
+package object gws {
+
+  export coulomb.units.accepted.{Millibar, ctx_unit_Millibar}
+
+  final type Bar
```
Comment (cquiroz) [inline modules/server_new/src/main/scala/observe/server/gws/package.scala:10]: We have some of these in core. I suggest we move these there

File: modules/server_new/src/main/scala/observe/server/odb/OdbProxy.scala [hunk 2]
```diff
@@ -140,8 +143,15 @@ object OdbProxy {
...
       false.pure[F]
 
-    override def datasetStartExposure(obsId: Observation.Id, fileId: ImageFileId): F[Boolean] =
-      true.pure[F]
+    override def datasetStartExposure(
+      obsId:  Observation.Id,
+      fileId: ImageFileId
+    ): F[RecordDatasetMutation.Data.RecordDataset.Dataset] =
+      Sync[F]
+        .delay(scala.util.Random.between(1L, Long.MaxValue))
```
Comment (cquiroz) [inline modules/server_new/src/main/scala/observe/server/odb/OdbProxy.scala:151]: You can use
https://typelevel.org/cats-effect/docs/std/random

File: modules/server_new/src/main/scala/observe/server/gws/GwsKeywordReader.scala [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import observe.server.EpicsHealth
+import observe.server.keywords.*
+
+object DewPoint extends NewType[Temperature[Double, Celsius]]
+type DewPoint = DewPoint.Type
+
+trait GwsKeywordReader[F[_]] {
+  def health: F[EpicsHealth]
+
+  def temperature: F[Temperature[Double, Celsius]]
```
[the remark this answers is not in the document]
Comment (jluhrs) [reply modules/server_new/src/main/scala/observe/server/gws/GwsKeywordReader.scala:24]: No, this is the ambient temperature. The next function is the one used to read the dew point.

File: modules/server_new/src/main/scala/observe/server/gws/package.scala (hunk 1 above)
Comment (jluhrs) [reply modules/server_new/src/main/scala/observe/server/gws/package.scala:10]: Where else would those be used?
I guess that some reporting tools will show weather conditions

File: modules/server_new/src/main/scala/observe/server/odb/OdbProxy.scala (hunk 2 above)
Comment (jluhrs) [reply modules/server_new/src/main/scala/observe/server/odb/OdbProxy.scala:151]: I tried after reading this comment, but for this particular use the current solution is simpler.

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**